psi4/driver/driver.py

```diff
@@ -1798,6 +1800,32 @@ def frequency(name, **kwargs):
 
 
 def vibanal_wfn(wfn, hess=None, irrep=None, molecule=None, project_trans=True, project_rot=True):
+    """Function to perform analysis of a hessian or hessian block, specifically...
+    calling for and printing vibrational and thermochemical analysis, setting thermochemical variables,
+    and writing the vibrec and normal mode files.
+
+    Parameters
+    ----------
+    wfn : psi4.core.wavefunction
+        The wavefunction which had its Hessian computed.
+    hess : psi4.core.Matrix
+        The hessian to analyze, if not the hessian in wfn.
+    irrep : int or string
+        The irrep for which frequencies are calculated. Thermochemical analysis is skipped if this is given,
+        as only one symmetry block of the hessian has been computed.
+    molecule : psi4.core.molecule or qcdb.molecule
+        The molecule to pull information from, if not the molecule in wfn. Must at least have similar
+        geometry to the molecule in wfn.
+    project_trans : boolean
+        Should translations be projected in the harmonic analysis?
+    project_rot : boolean
+        Should rotations be projected in the harmonic analysis?
+
+    Returns
+    -------
+    vibinfo : dict
+        A dictionary of vibrational information. See psi4/driver/qcdb/vib.py:harmonic_analysis
+    """
```

Comment:
Ah, thank you! I was just thinking today that this needed a docstring. A few changes:

```
wfn : :py:class:`~psi4.core.Wavefunction`
```

```
hess : ndarray of float, optional
    Hessian to analyze, if not the hessian in wfn.
    (3*nat, 3*nat) non-mass-weighted Hessian in atomic units, [Eh/a0/a0].
```
```
molecule : :py:class:`~psi4.core.Molecule` or qcdb.Molecule, optional
```
```
See :py:func:`~psi4.driver.qcdb.vib.harmonic_analysis`
```